sanic/config.py

```diff
@@ -201,7 +201,11 @@ def load_environment_vars(self):
         for k, v in os.environ.items():
             if k.startswith(SANIC_PREFIX):
                 _, config_key = k.split(SANIC_PREFIX, 1)
-                if v.isdigit():
-                    self[config_key] = int(v)
+                # This is a float or an int
```

Comment:
hmm...` try/except` is much more expensive than `if`check. it may be better to use `if` check.